### PR TITLE
Cover xml winevents t1021.002 executable file written in administrative smb share

### PR DESCRIPTION
**Pull request by mikeradka**
# PR Template for new Detections

For Authors:
- [ ] Make sure that CI/CD [detection-testing and build-and-validate](https://github.com/splunk/security_content/actions) jobs passed ✔️. 

For Reviewers:
- [ ] Verify CI/CD jobs have passed without errors.
- [ ] Validate SPL logic.
- [ ] Validate tags, description, and how to implement.
- [ ] Validate name matches `<platform>_<mitre att&ck technique>_<short description>`
- [ ] Verify references match analytic.
- [ ] Is there an Atomic Test?

### CODE REVIEW

File: detections/endpoint/executable_file_written_in_administrative_smb_share.yml [hunk 1]
```diff
@@ -12,11 +12,22 @@ description: The following analytic identifies executable files (.exe or .dll) b
   endpoints. Red Teams and adversaries alike may abuse administrative shares for lateral
   movement and remote code execution. The Trickbot malware family also implements
   this behavior to try to infect other machines in the infected network.
-search: '`wineventlog_security` EventCode=5145 Relative_Target_Name IN ("*.exe","*.dll")
-  Object_Type=File Share_Name IN ("\\\\*\\C$","\\\\*\\IPC$","\\\\*\\admin$") Access_Mask=
-  "0x2" | stats min(_time) as firstTime max(_time) as lastTime count by EventCode
-  Share_Name Relative_Target_Name Object_Type Access_Mask user src_port Source_Address
-  | `security_content_ctime(firstTime)` | `security_content_ctime(lastTime)` | `executable_file_written_in_administrative_smb_share_filter`'
+search: '`wineventlog_security` 
+    EventCode=5145
```
Comment (josehelps) [inline detections/endpoint/executable_file_written_in_administrative_smb_share.yml]: looks like there is an extra newline char here.

**Review by josehelps — CHANGES_REQUESTED** [review]
This looks good to me, only minor thing

File: detections/endpoint/executable_file_written_in_administrative_smb_share.yml (hunk 1 above)
Comment (mikeradka) [reply detections/endpoint/executable_file_written_in_administrative_smb_share.yml]: Ah, added that extra line for ease of readability so it looks clean when copy-pasta'd. Force of habit :)

image

Comment (mikeradka) [reply detections/endpoint/executable_file_written_in_administrative_smb_share.yml]: Went ahead and removed the newline char, so it looks like this when copy/pasted:

image

**Comment by josehelps** [comment]
Hey @mr-splunk we chatted about this in a call earlier last week, in short we should not handle the logic for TAs in the detection although super important you raised this. For now I will close this PR and not merge since we will push this fixes down the pike.

**Pull request closed without merging**